prime-router/src/main/kotlin/azure/FHIRFlowFunctions.kt

```diff
@@ -16,6 +26,38 @@ const val fhirQueueName = "fhir-raw-received"
  * Process will work through all the reports waiting in the 'process' queue
  */
 class FHIRFlowFunctions : Logging {
+    /**
+     * An azure function for processing an HL7 message into of FHIR
+     */
+    @FunctionName("convert")
+    @StorageAccount("AzureWebJobsStorage")
+    fun convert(
+        @HttpTrigger(
+            name = "convert",
+            methods = [HttpMethod.POST],
+            authLevel = AuthorizationLevel.FUNCTION,
+            route = "\$convert-data"
+        ) request: HttpRequestMessage<String?>,
+    ): HttpResponseMessage {
+        val responseBuilder = request.createResponseBuilder(HttpStatus.BAD_REQUEST)
```

Comment:
No security?  How are we protecting this?

Reply:
It's using the Azure function key auth for now